schema/common/defaults/configuration/roles-enabled.yml

```
@@ -0,0 +1,54 @@
+---
+kind: configuration/roles-enabled
```

```suggestion
kind: configuration/roles
```
We have `enabled` attribute so roles in this doc can be not only enabled but disabled as well. Thus my proposal to change kind name for generic.